src/Http/Http.Abstractions/src/Internal/ParsingHelpers.cs

```diff
@@ -14,11 +14,11 @@ public static StringValues GetHeader(IHeaderDictionary headers, string key)
         return headers.TryGetValue(key, out value) ? value : StringValues.Empty;
     }
 
-    public static StringValues GetHeaderSplit(IHeaderDictionary headers, string key)
+    public static string[] GetHeaderSplit(IHeaderDictionary headers, string key)
     {
         var values = GetHeaderUnmodified(headers, key);
 
-        StringValues result = default;
+        ValueListBuilder<string> strings = new([null!, null!, null!, null!]);
```

Comment:
I think this means "pre-allocate four slots" so that no further allocation will be required if there are four or fewer `Add` calls?

Reply:
yup